Makefile

```diff
@@ -161,11 +162,12 @@ local-gadget-tests:
 	rm -f ./local-gadget-manager.test
 
 .PHONY: integration-tests
-integration-tests: kubectl-gadget
+integration-tests:
 	KUBECTL_GADGET="$(shell pwd)/kubectl-gadget" \
 		go test ./integration/... \
 			-integration \
-			-image $(CONTAINER_REPO):$(IMAGE_TAG)
```

Comment:
I will add it back. Make sense to keep it to allow different image.

Reply:
This can now be handled via `INTEGRATION_TESTS_PARAMS` like [here](https://github.com/kinvolk/inspektor-gadget/pull/955/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R165).

Reply:
I think `-image` should be managed outside `INTEGRATION_TESTS_PARAMS` as it can be configured using `CONTAINER_REPO` and `IMAGE_TAG`. My idea with `INTEGRATION_TESTS_PARAMS` is more for debugging, when one wants to run just some tests `-run`, or multiple time `-count` or do not deploy the IG or SOP.

cc @eiffel-fl

Reply:
Yeah, makes sense. I will update the PR.